crates/evm/evm/src/inspectors/trace.rs

```diff
@@ -293,7 +293,7 @@ impl<DB: Database> InspectorExt<DB> for TraceCollector {
                         CreateOutcome {
                             result: InterpreterResult {
                                 result: InstructionResult::Return,
-                                output: Bytes::from(call.to.to_h256().0),
+                                output: Bytes::from(call.output),
```

Comment:
We can remove or update the comment above right? This should no longer be `call.to` but the bytecode we inserted con the call